contracts/script/GenerateAlloc.s.sol

```diff
@@ -159,37 +159,33 @@ contract GenerateAlloc is Script {
         vm.resetNonce(tmp);
 
         InitializableHelper.disableInitializers(impl);
-        IPTokenStaking(Predeploys.Staking).initialize(protocolAdmin, 1 ether, 1 ether, 1 ether, 7 days);
+        IIPTokenStaking.InitializerArgs memory args = IIPTokenStaking.InitializerArgs({
+            owner: protocolAdmin,
+            minStakeAmount: 1 ether,
+            minUnstakeAmount: 1 ether,
+            minCommissionRate: 5_00, // 5% in basis points
+            shortStakingPeriod: 1 days, // TBD
+            mediumStakingPeriod: 2 days, // TBD
+            longStakingPeriod: 3 days, // TBD
```

Comment:
Can this be updated in the staking contract after initialization?

Reply:
Yes, we can:
```
function setStakingPeriods(uint32 short, uint32 medium, uint32 long) external onlyOwner
```